readme.md

```diff
@@ -182,9 +214,56 @@ export default () => (
 
 ### Dynamic styles
 
+To make a component's visual representation customizable from the outside world, there are three options.
+
+#### Via interpolated dynamic props
+
+Any value that comes from the component's Render scope is treated as dynamic. This makes it possible to use `props` and `state` for example.
+
+```jsx
+const Button = (props) => (
+  <button>
+     { props.children }
+     <style jsx>{`
+        button {
+          padding: ${ 'large' in props ? '50' : '20' }px;
+          background: ${props.theme.background};
+          color: #999;
+          display: inline-block;
+          font-size: 1em;
+        }
+     `}</style>
+  </button>
+)
+```
+
+New styles injection is optimized to perform well at runtime and maintain 60fps.
+Nonetheless we recommend to split static and dynamic styles when possible. The following will be even faster:
```

Comment:
Is this because we need some dynamic computations ? It might be nice if there is a explanation a bit.

Reply:
If most of the CSS is static it would be ideal to use two `style` tags so that only the dynamic parts are recomputed/rendered. In fact when in dev mode we could print some message / warning when we detect similar situations (maybe in a followup)